src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,86 @@
+package kitchenpos.application;
+
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuGroupRepository;
+import kitchenpos.fixture.MenuGroupFixture;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatException;
+
+@SpringBootTest
+class MenuGroupServiceTest {
+
+    @Autowired
+    private MenuGroupService menuGroupService;
+
+    @Autowired
+    private MenuGroupRepository menuGroupRepository;
+
+    @Nested
+    @DisplayName("메뉴 그룹 생성")
+    class CreateGroupMenu {
+
+        @Test
+        @DisplayName("메뉴 그룹 생성 성공")
+        void testCreateGroupMenu() {
+            // given
+            final MenuGroup request = MenuGroupFixture.createMenuGroup("한마리메뉴");
+
+            // when
+            final MenuGroup result = menuGroupService.create(request);
+
+            // then
+            final MenuGroup found = menuGroupRepository.findById(result.getId()).orElse(null);
+
+            assertThat(found).isNotNull();
+            assertThat(found.getName()).isEqualTo(request.getName());
+        }
+
+        @ParameterizedTest
+        @NullAndEmptySource
+        @DisplayName("메뉴 그룹은 이름을 필수로 가진다.")
+        void testNullOrEmptyName(final String name) {
+            // given
+            final MenuGroup request = MenuGroupFixture.createMenuGroup(name);
+
+            // when & then
+            assertThatException()
+                    .isThrownBy(() -> menuGroupService.create(request))
+                    .isInstanceOf(IllegalArgumentException.class);
+        }
+    }
+
+    @Nested
+    @DisplayName("그룹 메뉴 조회")
+    class FindGroupMenus {
+
+        @Test
+        @DisplayName("등록된 모든 메뉴 그룹을 조회한다.")
+        void testFindAllGroupMenu() {
+            // given
+            MenuGroup menuGroup1 = MenuGroupFixture.createMenuGroup("한마리메뉴");
+            MenuGroup menuGroup2 = MenuGroupFixture.createMenuGroup("두마리메뉴");
```

Comment:
테스트 코드의 변수명에 넘버링보다 한글변수명등을 사용하면 직관적이 되어 가독성이 향상될 수 도 있어요. 😄 
```suggestion
            MenuGroup 한마리_메뉴그룹 = MenuGroupFixture.createMenuGroup("한마리메뉴");
            MenuGroup 두마리_메뉴그룹 = MenuGroupFixture.createMenuGroup("두마리메뉴");
```